config/agents/app/rbac/role.yaml

```diff
@@ -75,3 +75,12 @@ rules:
   - servicebindings/finalizers
   verbs:
   - update
+- apiGroups:
+  - primaza.io
+  resources:
+  - servicecatalogs
+  verbs:
+  - get
+  - list
+  - update
+  - watch
```

Comment:
this should not be needed. As of now, the agent don't need permissions on ServiceCatalogs. Right?

```suggestion
```

Reply:
done